lib/Fmt_ast.ml

```diff
@@ -556,6 +556,9 @@ let fmt_layout_str ~c ~loc string =
 
 let fmt_layout c l = fmt_layout_str ~c ~loc:l.loc (layout_to_string l.txt)
 
+let dummy_position ~loc =
+  Ast_helper.Exp.ident {loc; txt= Ldot (Lident "Lexing", "dummy_pos")}
```

Comment:
We have the location here.... maybe just make an actual correct location?

Reply:
Marking this question as resolved as discussed in slack.

For readers, introducing a string that was not in the original file is tricky in ocamlformat because ocamlformat eventually `String.sub`'s the original file with the location of the string, and since the inserted string (the pos_fname field in the Lexing.position record) was not in the original file, picking a location for `String.sub` is tricky.